contrastscan-remote/src/main/java/com/contrastsecurity/scan/GitHubActionCli.kt

```diff
@@ -0,0 +1,115 @@
+package com.contrastsecurity.scan
+
+import kotlin.Throws
+import kotlin.jvm.JvmStatic
+import picocli.CommandLine
+import java.io.File
+import java.lang.Exception
+import java.lang.IllegalArgumentException
+import java.net.URL
+import java.nio.file.Path
+import java.time.Duration
+import java.util.concurrent.Callable
+
+@CommandLine.Command(
+    name = "contrastscan-remote",
+    mixinStandardHelpOptions = true,
+    description = ["Perform contrastscan in a github action"]
+)
+class GitHubActionCli : Callable<Int> {
+    @CommandLine.Option(
+        names = ["-d", "--src-dir"],
+        defaultValue = ".",
+        description = ["where to find the source tree of the project"]
+    )
+    private lateinit var sourceRoot: File
+
+    @CommandLine.Option(
+        names = ["-m", "--prescan-out"],
+        defaultValue = "../prescan.json",
+        description = ["where to write the result (\${DEFAULT-VALUE})"]
+    )
+    private lateinit var prescanOutPath: File
+
+    @CommandLine.Option(
+        names = ["-s", "--sarif-out"],
+        defaultValue = "../results.sarif",
+        description = ["where to write the result (\${DEFAULT-VALUE})"]
+    )
+    private lateinit var sarifOutPath: File
+
+    @CommandLine.Option(
+        names = ["-u", "--url"],
+        // TODO: change this to public site
+        defaultValue = "https://teamserver-harmony-dev.contsec.com/Contrast",
+        description = ["Contrast Platform API url (\${DEFAULT-VALUE})"]
+    )
+    private lateinit var apiUrl: String
+
+    @CommandLine.Parameters(index= "0")
+    private lateinit var artifactPath: Path
+
+
+    @Throws(Exception::class)
```

Comment:
is this necessary?

Reply:
The editor added it for some reason.  So I just went along with it not really sure what it was saying.  I'll remove it.